packages/wallets/src/features/accounts/modules/DocumentService/components/Onfido/Onfido.tsx

```diff
@@ -39,7 +41,13 @@ const Onfido: React.FC<TOnfidoProps> = ({ onCompletion }) => {
     if (isLoading) return <Loader />;
 
     return (
-        <ModalStepWrapper title={localize('Add a real MT5 account')}>
+        <ModalStepWrapper disableAnimation={!!onClickBack} title={localize('Add a real MT5 account')}>
+            {onClickBack && (
+                <button className='wallets-onfido__back-button' onClick={onClickBack}>
+                    <LegacyArrowLeft2pxIcon iconSize='xs' />
+                    <Text weight='bold'>Back</Text>
+                </button>
```

Comment:
Just wondering why are we not using Button from `deriv-com/ui`

Reply:
Hi @jim-deriv,
This is the design for it.
<img width="1098" alt="Screenshot 2024-08-27 at 12 29 15 PM" src="https://github.com/user-attachments/assets/ce61cd9f-dff1-4072-b03c-e467f941af47">

Reply:
Btw it's Aizad not Jim 💀